os/tools/mkbinheader.py

```diff
@@ -201,15 +201,14 @@ def make_kernel_binary_header():
 
 def make_user_binary_header():
     binary_format = sys.argv[3]
-    kernel_ver = sys.argv[4]
-    binary_name = sys.argv[5]
-    binary_ver = sys.argv[6]
-    dynamic_ram_size = sys.argv[7]
-    main_stack_size = sys.argv[8]
-    main_priority = sys.argv[9]
-    comp_enabled = sys.argv[10]
-    comp_blk_size = sys.argv[11]
-    loading_priority = sys.argv[12]
+    binary_name = sys.argv[4]
```

Comment:
I think we need to update descriptions in above comments together, line 190~198. Please update them.

Reply:
Done.